docs/source/deployment/databricks.md

```diff
@@ -263,3 +263,9 @@ Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line mag
 ```ipython
 In [2]: %run_viz
 ```
+
+### How to use Datasets stored on Databricks DBFS
```

Comment:
A small thing, but this header is on the same level as the 1..10 steps that come before, and I understand this is not necessarily tied to those. Should we make it a second level heading? cc @stichbury

Reply:
Yes, let's do that. I was in two minds, but it makes sense to make it H2.